website/content/docs/getting-started/connect-to-target.mdx

```diff
@@ -180,5 +180,5 @@ details.
 
 ## Next Steps
 
-See our [common workflows](/docs/common-workflows) for in depth discussion on managing scopes, targets,
+See our [common workflows](https://learn.hashicorp.com/collections/boundary/common-workflows) for in depth discussion on managing scopes, targets,
```

Comment:
note: when clicking on this link it currently returns a `404`. Looking at the other link to `common-workflows` on the left side navbar, this seems the most appropriate.